FireMotD

```diff
@@ -159,6 +159,22 @@ VersionComparison () {
   return 3
 }
 
+function IsInstalled {
+  if [[ -x "/usr/bin/yum" ]] ; then
+    YumExec=$(which yum 2>/dev/null)
+    if yum list installed "$@" >/dev/null 2>&1; then
+     true
+    else
+      false
+    fi
+  elif [[ -x "/usr/bin/zypper" ]] ; then
+    WriteLog Verbose Info "No $@ check for zypper available."
+    true
+  elif [[ -x "/usr/bin/apt-get" ]] ; then
+    WriteLog Verbose Info "No $@ check for apt available."
```

Comment:
```
if dpkg-query -l "$@" >/dev/null 2>&1; then echo "true"; else echo "false"; fi
```

tested for moreutils, sysstat and openssh-client. Not sure why this is still needed though.

Reply:
If I remember correctly moreutils is needed for sponge, sysstat is need for mpstat. openssh-client I'm not sure anymore.. I'll leave that one out for now. Tx for the dpkg query I'll try integrate it